From cf4f218f42c2a43a520eac4f3a6251d557dcbe80 Mon Sep 17 00:00:00 2001
From: Michael Tremer <michael.tremer@ipfire.org>
Date: Tue, 17 Apr 2012 17:36:52 +0200
Subject: [PATCH 2/8] find-provides: Accept bogous soname.

This is a kind of weird bug. Some libs are "versioned" by
using their SONAME for symbol maps. This caused trouble
because the SONAME was not added to the list of provides
which is now changed by this commit.
---
 tools/find-provides |    1 -
 1 files changed, 0 insertions(+), 1 deletions(-)

diff --git a/tools/find-provides b/tools/find-provides
index 2e138ba..06b3c47 100755
--- a/tools/find-provides
+++ b/tools/find-provides
@@ -69,7 +69,6 @@ for file in ${binary_files}; do
 		/Version definitions:/ { START=1; }
 		/^[0-9]/ && (START==1) { print $4; }
 		/^$/ { START=0; }' | \
-                grep -v ${soname} | \
                 while read symbol ; do
                     echo "${soname}(${symbol})$(${is_64} && echo ${mark64} | sed 's/()//')"
                 done
-- 
1.7.7.6

